xml/System/Index.xml

```diff
@@ -138,9 +161,9 @@
         <Parameter Name="value" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">An object to compare with this Index object.</param>
+        <summary>Indicates whether the current Index object is equal to the passed object.</summary>
+        <returns><see langword="true" /> if the current Index object is equal to the passed object, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="value" /> is of type <see cref="T:System.Index" /> and is equal to the current instance; <see langword="false" /> otherwise.</returns>
```